src/app/mail/mail.component.scss

```diff
@@ -1239,7 +1240,7 @@
       .mail-sent-date{
         position: absolute;
         top: 17px;
-        right: 40px;
+        right: 1px;
```

Comment:
@angelo510 are you sure that you changed this value for mobile size only?

Reply:
And add one value for .task-danger

Reply:
Please review my commit carefully